src/js/components/awardv2/idv/activity/IdvActivityVisualization.jsx

```diff
@@ -63,6 +63,11 @@ export default class IdvActivityVisualization extends React.Component {
                 height={height}
                 width={this.state.visualizationWidth} />
         );
+        const resultsText = (
+            <div className="pagination__totals">
+                Displaying award orders <strong>{this.props.page}-{this.props.count}</strong> of {formatNumberWithPrecision(this.props.total, 0)}
```

Comment:
Additional hotfix needed to calculate the correct range of results. This is displayingthe  current page number - total number of results, when it should show the range of results for the current page.